logstash-core/src/main/java/org/logstash/KeyNode.java

```diff
@@ -60,8 +60,8 @@ private static String toString(Object value, String delim) {
         if (value == null) return "";
         if (value instanceof List) return join((List)value, delim);
         if (value instanceof BiValue) {
-            return ((BiValue) value).toString();
+            return value.toString();
```

Comment:
will this print the BiValue#toString , or just the Object#toString when removing the explicit cast ?